src/drop-down-menu.jsx

```diff
@@ -343,7 +359,12 @@ const DropDownMenu = React.createClass({
   },
 
   _onMenuRequestClose() {
-    this.setState({open:false});
+    setTimeout(() => {
```

Comment:
Without it, when you `requestClose('offScreen')` from the popover, the setState triggers some infinite loop type re-rendering issue which hangs the browser.  I only noticed it today - I think the core issue might be with the popover, but haven't been able to locate it.